### PR TITLE
reference/tools: add warning for deprecated loader

### PR DESCRIPTION
**Pull request by tennix**
### What is changed, added or deleted? (Required)

Loader now is deprecated and not maintained anymore, so add a warning to suggest user switch to TiDB Lightning.

Note: this PR also changes the loader.md file encoding from windows format to unix format, that is the new line is changed from CRLF to LF. See here for details (https://stackoverflow.com/questions/1552749/difference-between-cr-lf-lf-and-cr-line-break-types)

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: reference/tools/loader.md [hunk 1]
```diff
@@ -1,154 +1,158 @@
...
+---
+title: Loader 使用文档
+category: reference
+---
+
+# Loader 使用文档
+
+**警告：**
+
+> Loader 目前已经不再维护，其功能已经完全被 [TiDB Lightning TiDB backend 功能]取代，强烈建议切换到 TiDB Lightning。
```
Comment (kennytm) [inline reference/tools/loader.md:10]: this link points nowhere.

File: reference/tools/loader.md [hunk 2]
```diff
@@ -1,154 +1,158 @@
...
-target-table = "table"
-```
+---
+title: Loader 使用文档
+category: reference
+---
+
+# Loader 使用文档
+
+**警告：**
```
Comment (lilin90) [inline reference/tools/loader.md:8]:
```suggestion
> **警告：**
```

File: reference/tools/loader.md [hunk 3]
```diff
@@ -1,154 +1,158 @@
...
+
```
Comment (lilin90) [inline reference/tools/loader.md:9]:
```suggestion
>
```

**Comment by lilin90** [comment]
@kennytm @WangXiangUSTC PTAL~ Thanks!

**Comment by WangXiangUSTC** [comment]
LGTM

**Comment by lichunzhu** [comment]
related issue https://github.com/pingcap/tidb-lightning/issues/305

**Pull request merged**